src/printer/index.js

```diff
@@ -189,10 +150,29 @@ function printIf(path, print) {
     indent(concat([hardline, printBody(path, print)]))
```

Comment:
nit: If you wanted, you could update this function to use the `indentConcat` function, which reduces the number of parens you need :D